flight/targets/boards/discoveryf4bare/bootloader/Makefile

```diff
@@ -16,7 +16,7 @@
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 #
 
-ifndef OPENPILOT_IS_COOL
+ifndef LIBREPILOT_IS_COOL
```

Comment:
Good spotting. I think we would prefer to change this to something more vendor neutral though